### PR TITLE
Change README.md to standard structure

### PR DESCRIPTION
**Pull request by caldeirav**
As described, following standard contribution template and added a project logo as coolness factor.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,97 @@
...
+  <a href="https://github.com/opendatahub-io-contrib/genai-llm-rag-pattern">
+    <img src="images/Logo_RAG_LLM_Pattern.png" alt="Logo" width="160" height="160">
+  </a>
 
-Today a dominant usecase for deploying Large Language Models (LLMs) within the enteprise is as a enterprise search assistant. LLMs without any underlying context a prone to halucination. Retrieval Augmented Generation (RAG) is an architecture where the LLM is pared with a vector database. The vector database contains a specialised index of enterprise documents which provide 'context' to the LLM allowing more accurate results AND allowing employees to follow up with source documents. This project is a reference pattern for deploying a RAG search assistant built on top of Red Hat OpenShift AI.
+  <h3 align="center">Validated Pattern for Retrieval-Augmented Generation (RAG) with LLM</h3>
+  
+  <p align="center">
+    <br />
+    <a href="https://github.com/opendatahub-io-contrib/genai-llm-rag-pattern"><strong>Explore the docs »</strong></a>
```
Comment (butler54) [inline README.md]: This link should be to here `https://opendatahub-io-contrib.github.io/genai-llm-rag-pattern/`

File: README.md [hunk 2]
```diff
@@ -1,22 +1,97 @@
...
+## License
+
+Distributed under the Apache-2.0 License. See `LICENSE.txt` for more information.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- MARKDOWN LINKS & IMAGES -->
+<!-- https://www.markdownguide.org/basic-syntax/#reference-style-links -->
+[product-screenshot]: images/screenshot.png
```
Comment (butler54) [inline README.md]: This is unused.

File: README.md [hunk 3]
```diff
@@ -1,22 +1,97 @@
...
+3. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
+4. Push to the Branch (`git push origin feature/AmazingFeature`)
+5. Open a Pull Request
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+## License
+
+Distributed under the Apache-2.0 License. See `LICENSE.txt` for more information.
```
Comment (butler54) [inline README.md]: Should be `LICENSE` not `LICENSE.txt`

File: README.md [hunk 4]
```diff
@@ -1,22 +1,97 @@
...
+  <h3 align="center">Validated Pattern for Retrieval-Augmented Generation (RAG) with LLM</h3>
+  
+  <p align="center">
+    <br />
+    <a href="https://github.com/opendatahub-io-contrib/genai-llm-rag-pattern"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/opendatahub-io-contrib/genai-llm-rag-pattern">View Demo</a>
+    ·
+    <a href="https://github.com/opendatahub-io-contrib/genai-llm-rag-pattern/issues/new?labels=bug&template=bug-report---.md">Report Bug</a>
```
Comment (butler54) [inline README.md]: Templates don't exist yet for bug / feature requests.

File: README.md (hunk 1 above)
Comment (caldeirav) [reply README.md]: Corrected

File: README.md (hunk 2 above)
Comment (caldeirav) [reply README.md]: Removed in the new commit

File: README.md (hunk 3 above)
Comment (caldeirav) [reply README.md]: Corrected

File: README.md (hunk 4 above)
Comment (caldeirav) [reply README.md]: Use the main link until we create suitable templates

**Review by bryonbaker — APPROVED** [review]

**Pull request merged**